pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/kdefault/enablecainection_patch.go

```diff
@@ -49,13 +50,27 @@ const injectCAPatchTemplate = `# This patch add annotation to admission webhook
 apiVersion: admissionregistration.k8s.io/{{ .Resource.Webhooks.WebhookVersion }}
 kind: MutatingWebhookConfiguration
 metadata:
+  labels:
+    app.kubernetes.io/name: mutatingwebhookconfiguration
+    app.kubernetes.io/instance: mutating-webhook-configuration
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: k8s.io
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02